docs/index.md

```diff
@@ -47,7 +45,7 @@ ARGUMENTS
     
     trailing arguments:
      transformation (required)   The type of transformation used: simple, thematische-collectie,
-                                 original-versioned, fedora-versioned.
+                                 original-versioned.
```

Comment:
`thematische-collectie` is also no longer relevant (never implemented i think)

Reply:
Datasets are rejected or not.

https://github.com/DANS-KNAW/easy-fedora-to-bag/blob/6d9121d0f5f51a84b5890a538100196659b93ec9/src/main/scala/nl/knaw/dans/easy/fedoratobag/Command.scala#L51
https://github.com/DANS-KNAW/easy-fedora-to-bag/blob/8ef3a0bade9a48444808ae24d24b0a53147ef13c/src/main/scala/nl/knaw/dans/easy/fedoratobag/filter/SimpleDatasetFilter.scala#L22